docs/schema/method-element-in-methods-bdcmetadata-schema.md

```diff
@@ -31,114 +34,30 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Attributes
 
-<table>
-<colgroup>
-<col width="20%" />
-<col width="80%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th align="left"><p>Attribute</p></th>
-<th align="left"><p>Description</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td align="left"><p>**IsStatic**</p></td>
-<td align="left"><p>Optional.</p>
-<p>Specifies whether the execution of this method requires an external item (**EntityInstance**) to serve as a context for execution. If set to **true**, the method represents a static method and does not require a specific **EntityInstance** to provide context for execution. If set to **false**, the method represents an instance method and requires an **EntityInstance** to provide the context for execution.</p>
-<p>Default value: **true**</p>
-<p>Attribute type: **Boolean**</p></td>
-</tr>
-<tr class="even">
-<td align="left"><p>**LobName**</p></td>
-<td align="left"><p>Optional.</p>
-<p>The name of the operation defined in the external system that is represented by this method.</p>
-<p>Attribute type: **String**</p></td>
-</tr>
-<tr class="odd">
-<td align="left"><p>**Name**</p></td>
-<td align="left"><p>Required.</p>
-<p>The name of this method.</p>
-<p>Attribute type: **String**</p></td>
-</tr>
-<tr class="even">
-<td align="left"><p>**DefaultDisplayName**</p></td>
-<td align="left"><p>Optional.</p>
-<p>The default display name of the method.</p>
-<p>Attribute type: **String**</p></td>
-</tr>
-<tr class="odd">
-<td align="left"><p>**IsCached**</p></td>
-<td align="left"><p>Optional.</p>
-<p>Specifies whether this method is used frequently. If set to **true**, Business Data Connectivity (BDC) service caches this method in memory.</p>
-<p>Default value: **true**</p>
-<p>Attribute type: **Boolean**</p></td>
-</tr>
-</tbody>
-</table>
+|Attribute|Description|
+|---------|-----------|
+|**IsStatic**|Optional.<br /><br />Specifies whether the execution of this method requires an external item (**EntityInstance**) to serve as a context for execution. If set to **true**, the method represents a static method and does not require a specific **EntityInstance** to provide context for execution. If set to **false**, the method represents an instance method and requires an **EntityInstance** to provide the context for execution.<br /><br />Default value: **true**<br /><br />Attribute type: **Boolean**
```

Comment:
Need to leave pipes at the end of lines 39-43.  There should be 3 pipes per line.  I do a search for "|" and it highlights all the pipes.